root/work/edit_form.tt

```diff
@@ -105,9 +105,54 @@
   </div>
 
   <div class="documentation">
-    [%- iswc_bubble(link_entity(work)) -%]
+    <div class="bubble" id="name-bubble">
+      <p>
+        [% l('The {doc|name} is the work’s canonical title in the language in which it was originally written.',
+             { doc => { href => doc_link('Work#Name'), target => '_blank' } }) %]
+      </p>
+    </div>
+
+    <div class="bubble" id="comment-bubble">
+      <p>
+        [% l('The {doc|disambiguation} field helps users distinguish between similarly-named works.',
+             { doc => { href => doc_link('Disambiguation_Comment'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please leave it blank if there are enough other differences between the works that they are unlikely to be confused.') %]
```

Comment:
Actually I just remembered for works we have actual guidelines for this: https://beta.musicbrainz.org/doc/Style/Work#Disambiguation_comment (it's not really that guideline-y but). Should we link it? In any case "Please leave it blank" does make sense in this specific case.

Reply:
Nice catch, and sure, makes sense to me. I've added another paragraph with the standard "Please see the style guidelines for more information." text locally. (Hopefully it won't look too strange to have two sentences in a row starting with "Please".)